Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -186,10 +192,22 @@ private void MutateInt(ref int val, int min, int max, int bits, int totalbits, f
         if (!Random(probBitflip))
             return;
 
+        if (min == max)
+        {
+            val = min;
+            return;
+        }
+
+        // Starting number of bits that are high, between 0 and bits.
+        // In other words, it's val mapped linearly from range [min, max] to range [0, bits], and then rounded.
+        int valInt = (int) MathF.Round((val - min) / (max - min) * bits);
+        // val may be outside the range of min/max due to starting prototype values, so clamp.
+        valInt = Math.Clamp(valInt, 0, bits);
```

Comment:
Please eliminate the space after the cast per the [style guidelines](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/types/casting-and-type-conversions). Also consider using `MathHelper.Lerp` which might convey the intent better.

Reply:
Fixed the spacing